code/datums/gamemode/factions/legacy_cult/hell_universe.dm

```diff
@@ -103,12 +103,14 @@ In short:
 			T.underlays += "hell01"
 		CHECK_TICK
 
+/*
 	for(var/datum/lighting_corner/C in global.all_lighting_corners)
 		if (!C.active)
 			continue
 
 		C.update_lumcount(0.5, 0, 0)
 		CHECK_TICK
+*/
```

Comment:
I do not recommend commenting out code with block comments. That was how we got MEGAMADCRAB